examples/dft/compile_time_dispatching/complex_fwd_buffer_mklgpu.cpp

```diff
@@ -43,80 +28,76 @@
 #endif
 #include "oneapi/mkl.hpp"
 
-// local includes
-#include "example_helper.hpp"
-
-void run_getrs_example(const sycl::device& cpu_device) {
-    // Matrix sizes and leading dimensions
-    constexpr std::size_t N = 10;
-    std::int64_t rs[3] {0, N, 1};
+void run_example(const sycl::device& gpu_deviceice) {
+    constexpr int N = 10;
 
-
-    // Catch asynchronous exceptions for cpu and cpu
-    auto cpu_error_handler = [&](sycl::exception_list exceptions) {
+    // Catch asynchronous exceptions for cpu
+    auto gpu_error_handler = [&](sycl::exception_list exceptions) {
         for (auto const& e : exceptions) {
             try {
                 std::rethrow_exception(e);
             }
             catch (sycl::exception const& e) {
                 // Handle not dft related exceptions that happened during asynchronous call
-                std::cerr
-                    << "Caught asynchronous SYCL exception on cpu device during GETRF or GETRS:"
-                    << std::endl;
+                std::cerr << "Caught asynchronous SYCL exception:" << std::endl;
                 std::cerr << "\t" << e.what() << std::endl;
             }
         }
         std::exit(2);
     };
 
-    std::cout << "DFTI example" << std::endl;
-    //
-    // Preparation on cpu
-    //
-    sycl::queue cpu_queue(cpu_device, cpu_error_handler);
-    sycl::context cpu_context = cpu_queue.get_context();
-    sycl::event cpu_getrf_done;
+    sycl::queue gpu_queue(gpu_deviceice, gpu_error_handler);
 
-    double *x_usm = (double*) malloc_shared(N*2*sizeof(double), cpu_queue.get_device(), cpu_queue.get_context());
+    std::vector<std::complex<float>> input_data(N);
+    std::vector<std::complex<float>> output_data(N);
 
     // enabling
-    // 1. create descriptors 
-    oneapi::mkl::dft::descriptor<oneapi::mkl::dft::precision::DOUBLE, oneapi::mkl::dft::domain::COMPLEX> desc(N);
+    // 1. create descriptors
+    oneapi::mkl::dft::descriptor<oneapi::mkl::dft::precision::SINGLE,
+                                 oneapi::mkl::dft::domain::COMPLEX>
+        desc(N);
 
     // 2. variadic set_value
-    desc.set_value(oneapi::mkl::dft::config_param::PLACEMENT, oneapi::mkl::dft::config_value::NOT_INPLACE);
-
-    // 3. commit_descriptor (compile_time CPU)
-    desc.commit(oneapi::mkl::backend_selector<oneapi::mkl::backend::mklcpu>{ cpu_queue });
-
-    // 5. compute_forward / compute_backward (CPU)
-    // oneapi::mkl::dft::compute_forward(desc, x_usm);
+    desc.set_value(oneapi::mkl::dft::config_param::PLACEMENT,
+                   oneapi::mkl::dft::config_value::NOT_INPLACE);
+    desc.set_value(oneapi::mkl::dft::config_param::NUMBER_OF_TRANSFORMS,
+                   static_cast<std::int64_t>(1));
+
+    // 3. commit_descriptor (compile_time MKLGPU)
+    desc.commit(oneapi::mkl::backend_selector<oneapi::mkl::backend::mklgpu>{ gpu_queue });
+
+    // 5. compute_forward / compute_backward (MKLGPU)
```

Comment:
```suggestion
    // 4. compute_forward / compute_backward (MKLGPU)
```
(or just remove the numbering)

Reply:
Done in [95f4e2c](https://github.com/hjabird/oneMKL/pull/6/commits/95f4e2ce326ac7a1c78068056ff9bbf3a400d705).